torch_xla/csrc/ops/index_ops.cpp

```diff
@@ -260,7 +260,9 @@ CanonicalIndexInfo GetCanonicalIndexInfo(
   CheckIndexTensorTypes(orig_indices);
   // First expand ByteTensor (boolean masks) into 1 or more LongTensors, then
   // broadcast all index tensors together.
-  auto indices = xla_expand_outplace(ExpandByteTensors(base, orig_indices));
+  std::vector<at::Tensor> expand_byte_tensors =
+      ExpandByteTensors(base, orig_indices);
+  std::vector<at::Tensor> indices = xla_expand_outplace(expand_byte_tensors);
```

Comment:
what's this change for?

Reply:
It's just a refactor, breaking one line into two.